### PR TITLE
feature(website): new landing page

### PR DESCRIPTION
**Pull request by ssandino**
wip: structure and content for now

### CODE REVIEW

**Comment by ssandino** [comment]
@DarkMenacer: this is how it looks. All content is on purpose without design (and also just the first layer, so to speak without the content for popups)

Screenshot 2024-02-21 at 00 30 58

**Comment by DarkMenacer** [comment]
@ssandino 
I've modified the introduction.tsx file to further simplify the translations...have a look and let me know what you feel about it

File: shared/locales/en/website-home2.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+				{ "text": "They just rarely have the opportunity – or the money – to do so. That’s where " },
+				{ "text": "you and Social Income", "color": "accent" },
+				{ "text": "come in." }
+			]
+		],
+		"subtitles": [
+			[{ "text": "Unique Process for Fair Selection" }],
+			[{ "text": "Direct Giving, Direct Results" }],
+			[{ "text": "Ongoing Support" }]
+		],
```
Comment (DarkMenacer) [inline shared/locales/en/website-home2.json:28]: Here are the changes that I've made

File: website/src/app/[lang]/[region]/(website)/home2/(sections)/introduction.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+			<div>{joinTranslation(titles[1])}</div>
+			{/*<Typography>{translator.t('section-2.subtitle-1')}</Typography>*/}
+			{joinTranslation(subtitles[0])}
+			<Typography>{translator.t('section-2.text-1.1')}</Typography>
+			<Typography>{translator.t('section-2.text-1.2')}</Typography>
+			{/*<Typography>{translator.t('section-2.subtitle-2')}</Typography>*/}
+			{joinTranslation(subtitles[1])}
+			<Typography>{translator.t('section-2.text-2')}</Typography>
+			{/*<Typography>{translator.t('section-2.subtitle-3')}</Typography>*/}
+			{joinTranslation(subtitles[2])}
```
Comment (DarkMenacer) [inline website/src/app/[lang]/[region]/(website)/home2/(sections)/introduction.tsx:45]: I find the code to be a lot more simplified this way

**Comment by ssandino** [comment]
Great. Thanks @DarkMenacer – we'll discuss tomorrow directly with @mkue

**Comment by DarkMenacer** [comment]
Creating a new branch from here on out to start with individual sections but not disturb this branch so that it can be merged (for the structure)

**Comment by DarkMenacer** [comment]
New branch with sections code: `pranav/new-landing-page`

**Comment by ssandino** [comment]
@DarkMenacer I changed the status of the PR to ready to be reviewed. @mkue thanks for reviewing this foundational branch for further development of the landing page and new design.

**Comment by mkue** [comment]
this looks great and is a very good starting point for the new landing page and new website layout. i moved the folder one layer up so that we can use a separate layout with a new theme, that i created as well, with some basic colors from the figma.

**Review by mkue — APPROVED** [review]

**Pull request merged**